README.md

```diff
@@ -114,6 +114,12 @@ use Mongo by using the jobDB, jobDBAddress, jobDBUsername, and jobDBPassword par
 kala run --jobDB=mongo --jobDBAddress=server1.example.com,server2.example.com --jobDBUsername=admin --jobDBPassword=password
 ```
 
+use Postgres by using the jobDB, jobDBAddress params:
+
+```bash
+kala run --jobDB=postgres --jobDBAddress=server1.example.com/kala?sslmode=disable --jobDBUsername=admin --jobDBPassword=password
```

Comment:
This is a total nitpick, but can we switch the `sslmode` to `sslmode=required`?

Reply:
👍